src/ExpressionEvaluator/CSharp/Source/ExpressionCompiler/Symbols/SimpleTypeParameterSymbol.cs

```diff
@@ -48,7 +48,7 @@ public override bool HasReferenceTypeConstraint
             get { return false; }
         }
 
-        internal override bool HasNullableReferenceTypeConstraint
+        internal override bool? ReferenceTypeConstraintIsNullable
         {
             get { return false; }
```

Comment:
>false [](start = 25, length = 5)

Should this depend on context?

Reply:
> Should this depend on context?

No, there is no ReferenceTypeConstraint

---
In reply to: [213865904](https://github.com/dotnet/roslyn/pull/29586#discussion_r213865904) [](ancestors = 213865904)

Reply:
Makes sense. Would be good to assert or throw, since we shouldn't call this. Thanks